src/RevPrefetcher.cc

```diff
@@ -64,6 +74,18 @@ bool RevPrefetcher::IsAvail(uint64_t Addr){
   return false;
 }
 
+void RevPrefetcher::MarkInstructionLoadComplete(const MemReq& req){
+  auto it = OutstandingFetchQ.begin();
+  while((it != OutstandingFetchQ.end())){
+    if(it->Addr == req.Addr){
+      OutstandingFetchQ.erase(it++);
+      break;
```

Comment:
Actually, since you're `break`ing and `it` isn't used afterwards, it is not necessary to increment `it` here.